providers/catalog.go

```diff
@@ -1526,6 +1527,32 @@ var catalog = CatalogType{ // nolint:gochecknoglobals
 			Write:     false,
 		},
 	},
+
+	// Faceboook Ads Manager Configuration
+	FacebookAdsManager: {
```

Comment:
Since this is the entire graph API, can you call this just `Facebook` and make sure to update the auth/token URLs if needed?